### DISCUSSION TITLE
How to load a tiled map

### DISCUSSION THREAD

**Discussion by hasikill** [axmolengine/axmol#2062] [Q&A] [answered]
I don't find any support for tiled in the extension. How can I use axmol to load tiled correctly?

**Comment by rh101** [comment, marked as answer]
> I don't find any support for tiled in the extension. How can I use axmol to load tiled correctly?

Take a look at the [cpp-tests](https://github.com/axmolengine/axmol/blob/dev/docs/DevSetup.md#how-to-quick-build-a-test-project-eg-cpp-tests) project.  It has a section for demonstrating tile maps in Axmol.  Build the project and review the source code to see how it is done.

**Reply by hasikill** [reply]
OK, got it. Thank you.
